src/views/ResourceInfo.js

```diff
@@ -0,0 +1,200 @@
+    type: ['Education Tool'],
+    path: ['Explorer Path'],
+    avatarIcon: "https://cdn.substack.com/image/fetch/w_170,c_limit,f_auto,q_auto:best,fl_progressive:steep/https%3A%2F%2Fbucketeer-e05bbc84-baa3-437e-9518-adb32be77984.s3.amazonaws.com%2Fpublic%2Fimages%2F3fcac70c-a37c-4152-a109-f702c6375c31_256x256.png",
+    aiSystemsType:[],
+    uploadDate: '',
+    creationDate: '',
+    modifiedDate:'',
+    licenseName :'',
+    technical: '',
+    trustIndexCategories: ['Explainability & Interpretability'],
+    fundedBy: '',
+    creator: '',
+    dataDictLink: '',
+    sensitiveData: '',
+    qualityReview: '',
+    ethicsReview: '',
+    usage : '',
+    isConfidential: '',
+    offensiveContent: '',
+    numInstances: '',
+    instances: [],
+    label: '', 
+    rawData: '',
+    distribution: '',
+    personalInfoRemoved: '',
+    privacyProcedure :'',
+    individualsIdentified : '',
+    noiseDescription :' ',
+    externalRestrictions:'',
+    files: [{name: "Dictionary", link: "https://montrealethics.ai/dictionary/", type:"URL"},
+  {name:"Click me", link:"www.google.com", type:"URL"}],
+    topics: ["Other topic"],
+    organizations: ["Montreal AI Ethics Institute"]
+  };
+
+function FileTable(props) {
+  const columns = [
+    {
+      title: 'Name',
+      dataIndex: 'name',
+      key: 'name',
+    },
+    {
+      title: 'Type',
+      dataIndex: 'type',
+      key: 'type',
+    },
+    {
+      title: 'Link',
+      dataIndex: 'link',
+      key: 'link',
+      render: text => <a href={text}>{text}</a>
+    },
+  ]
+  return (
+    <div>
+      <h1 style={{ padding:'10px',fontSize: '2em', fontWeight: 'bold'}}>Files</h1>
+      <Table columns={columns} dataSource={props.data} />
+    </div>
+  )
+}
+
+function SideBar(props){
+  return(
+  <Affix offsetTop={60}>
+    <Sider width={250}>
+      <Menu
+        mode="inline"
+        theme="light"
+        defaultOpenKeys={['overview']}
+        style={{ height: '100%', borderRight: 0 }}
+      >
+        <Menu.Item 
+          key="overview" 
+          icon={<FileDoneOutlined/>}
+          style={{ marginTop: '30px' }}
+          onClick={() => {
+            props.topRef.current.scrollIntoView();
+          }}>
+            Overview
+        </Menu.Item>
+
+        <Menu.Item 
+          key="details"  
+          icon={<SearchOutlined/>}                     
+          onClick={() => {
+            props.detailRef.current.scrollIntoView();
+          }} >
+            Details 
+        </Menu.Item>
+        <Menu.Item 
+          key="files"
+          icon={<FolderOpenOutlined/>}
+          onClick={() => {
+            props.fileRef.current.scrollIntoView();
+          }} >
+            Files 
+        </Menu.Item>
+      </Menu>
+    </Sider>
+  </Affix> );
+}
+
+
+export default function ShowExample() {
+  return (
+    <ResourceInfo resource = {resourceData}/>
+  );
+}
+
+function ResourceInfo(props) {
```

Comment:
"ViewResource". Also is what should be exported (you can prob delete ShowExample and just define the resource example data inside the component).